### PR TITLE
LPD-39537 - Stored XSS in Components portlet

### PR DESCRIPTION
**Pull request by brbalazs**
https://liferay.atlassian.net/browse/LPD-39537

### CODE REVIEW

**Comment by brbalazs** [comment]
ci:test:sf

File: modules/test/playwright/tests/client-extension-web/customElement.spec.ts [hunk 1]
```diff
@@ -4,16 +4,21 @@
  */
 
 import {Page, expect, mergeTests} from '@playwright/test';
+import assert from 'node:assert';
```
Comment (smottal) [inline modules/test/playwright/tests/client-extension-web/customElement.spec.ts:7]: ????

File: modules/test/playwright/tests/client-extension-web/customElement.spec.ts [hunk 2]
```diff
@@ -133,3 +138,39 @@ test(
...
+
+		await applicationsMenuPage.goToComponents();
+
+		await expect(componentsPage.helpLink).toBeVisible();
+
+		page.on('dialog', async (dialog) => {
+			if (dialog.type() === 'alert' && dialog.message() === '123') {
+				await dialog.dismiss();
+				assert.fail();
+			}
```
Comment (smottal) [inline modules/test/playwright/tests/client-extension-web/customElement.spec.ts:171]: we can remove this... if a dialog opens... test fails

File: modules/test/playwright/tests/client-extension-web/customElement.spec.ts [hunk 3]
```diff
@@ -133,3 +138,39 @@ test(
...
+		});
+
+		await page.reload();
```
Comment (smottal) [inline modules/test/playwright/tests/client-extension-web/customElement.spec.ts:174]: please add also a check that the NAME is printed somewhere in the page

**Review by smottal — CHANGES_REQUESTED** [review]

**Review by smottal — APPROVED** [review]

**Comment by smottal** [comment]
ci:test:relevant

**Comment by brbalazs** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**